pkg/namesgenerator/names-generator.go

```diff
@@ -186,6 +186,9 @@ var (
 		// Emmett Brown invented time travel. https://en.wikipedia.org/wiki/Emmett_Brown (thanks Brian Goff)
 		"brown",
 
+		// Michael J Burry - one of the first people to predict the economic fall of 2008
+		"burry",
```

Comment:
Bit in doubt if this one fits in with the general theme.

Given that we'd also like [Women and Men to be equally represented](https://github.com/moby/moby/blob/master/pkg/namesgenerator/names-generator.go#L106); perhaps you can add a different person instead?

Reply:
Sure thing!